GitExtUtils/ArgumentBuilder.cs

```diff
@@ -69,10 +68,12 @@ public void Add(string? s)
         /// <param name="args">The arguments to add to this builder.</param>
         public void AddRange(IEnumerable<string> args)
         {
-            args = args.Where(a => !string.IsNullOrEmpty(a));
             foreach (string s in args)
             {
-                Add(s);
+                if (!string.IsNullOrEmpty(s))
```

Comment:
This check is already done in `Add`.
Hence, it should be removed here at all. And the handling of empty arguments by `AddRange` should be verified by means of a new testcase in `ArgumentBuilderTests.cs`.